### DISCUSSION TITLE
Get email copy when sending to all customers (or selected group)

### DISCUSSION THREAD

**Discussion by holgerm** [pretix/pretix#4477] [Q&A]
We often send emails with further info in advance of an event to all (or certain) customers (last entry in left menu). But we never see these emails and if something goes wrong we do not see that easily. Is there an option to get a copy of that mails? Or do we need to make an additional internal "fake" order to get these mails too?

**Comment by pc-coholic** [comment]
That might be the easiest solution, if you only want those messages...

Else, you could also setup pretix to send you *all* outgoing messages in BCC - but that can be a lot of messages and - depending on your mailhoster - will get you on some list since you're basically spamming yourself with a bunch of very similar emails...

**Reply by holgerm** [reply]
What I would like to have is details about the success in the list of sent emails (in German "Email-Verlauf"). What I also miss there is a list of actual recipients (email adresses) as well as attachments.

**Reply by holgerm** [reply]
Currently I can check the success of sent emails under the menu (German:) "Kontoänderungen" which is not intuitiv for me ;-)
